model/feature-flags/registry.yaml

```diff
@@ -11,7 +11,7 @@ groups:
         stability: development
         brief: The lookup key of the feature flag.
         examples: ["logo-color"]
-      - id: feature_flag.provider_name
```

Comment:
you'd have to deprecate it, but add the annotation to exclude from codegen following example in https://github.com/open-telemetry/semantic-conventions/pull/2086

Reply:
@lmolkova done in feada1a98ca86286b5164d79cbfc66af78870c65 PTAL